### PR TITLE
Added pages: "Games", "About" and "Cards", and styles.

### PR DESCRIPTION
**Pull request by LotusElize**
In this commit, two new website pages were added:

1. "About":
   - Added the "About" page with relevant content.
   - Utilized Flexbox for styling and layout of the "About" page.

2. "Cards":
   - Added the "Cards" page with content and applied grids for organizing cards on the page.

This commit includes significant changes related to web development and introduces new pages with their respective functional elements.

### CODE REVIEW

File: eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    { SwitchCase: 1 },
+                ],
+                'no-cond-assign': ['error', 'always'],
+                'no-console': 'off',
+                'no-alert': 'off',
+                'func-names': 'off',
+            },
+        },
+    ],
+};
```
Comment (vladyslav-yermolin-moc) [inline eslintrc.js:158]: Hey, @LotusElize, please remove these files `.DS_Sore` from your pr 
image

File: eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
+module.exports = {
```
Comment (vladyslav-yermolin-moc) [inline eslintrc.js:1]: And this file should be removed also

File: homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html:1]: That is what I see, when open your page 
The typical files structure for static html site is this

- folder for styles
- folder for images
- index.html
- any_other_files.html

image

Comment (vladyslav-yermolin-moc) [reply homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html:1]: Please correct the file structure or update path to styles and relative page, I can't check your work now.
The index.html should contain about info, so no need to create for it separate html page

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html (hunk 3 above)
Comment (LotusElize) [reply homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html:1]: @vladyslav-yermolin-moc Thanks, done refactored , in new commit for this pull request  [2-css_LotusElize (refactored)](https://github.com/MastersAcademy/fe-course-basic-2023/pull/143/commits/592a5154278040c7f20b74900969a781bdcf5576)

**Comment by vladyslav-yermolin-moc** [comment]
So no I still see a lot of unnecessary files in your [changes](https://github.com/MastersAcademy/fe-course-basic-2023/pull/143/files). Please remove them 
image

**Comment by vladyslav-yermolin-moc** [comment]
I've recorded screencast for you
https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/907410b3-4c43-4aa5-b60c-bfb933674ed3

File: .gitignore [hunk 4]
```diff
@@ -64,3 +64,4 @@ typings/
 
 karma.conf.js
 protractor.conf.js
+.DS_Store
```
Comment (vladyslav-yermolin-moc) [inline .gitignore:67]: Hey, @LotusElize it is a nice move but in your task, you SHOULD work only in YOUR folder.
So, please remove the changes to the files `.gitignore` and `eslintrc.js` from your pull request. That is mean, you have to revert the files to their state in the main branch: 
- https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore
- https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore

File: homeworks/taras.bobov_LotusElize/GamesProject/games.html [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+                            <div class="card__header__title">
+                                <h2 class="card__title">Last os Us</h2>
+                                <p class="card__paragraph">
+                                    A post-apocalyptic tale of survival
+                                </p>
+                            </div>
+                        </div>
+                        <ul class="card__list">
+                            <li>
+                                <p>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/taras.bobov_LotusElize/GamesProject/games.html:50]: Why do you add this `p` here? It is useless

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
Nice pictures and shadows, BTW.

**Comment by vladyslav-yermolin-moc** [comment]
And you have some conflicts, please resolve them here, or merge them to your branch fresh main and resolve conflicts locally. Google or ask bots for help

File: .gitignore [hunk 6]
```diff
@@ -1,66 +0,0 @@
-# Logs
```
Comment (vladyslav-yermolin-moc) [inline .gitignore:1]: Nah, you should not delete this file, you should revet it to the original one in the main branch https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 6 above)
Comment (vladyslav-yermolin-moc) [reply .gitignore]: Well done👏

File: eslintrc.js (hunk 2 above)
Comment (vladyslav-yermolin-moc) [reply eslintrc.js:1]: Hey, @LotusElize, please revert this file to its original state as you've done for `.gitignore`

Comment (vladyslav-yermolin-moc) [reply eslintrc.js:1]: Your changes should be done in your folder only. Now it will change the `.eslint.js` for the whole project, if I merge your pr

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Review by vladyslav-yermolin-moc — APPROVED** [review]

File: homeworks/taras.bobov_LotusElize/GamesProject/css/games.css [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  height: 260px;
+  background-color: #ECECEC;
+  color: #221F1F;
+  display: grid;
+  padding: 12px 12px 22px;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  border-radius: 12px;
+  -webkit-border-radius: 12px;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/taras.bobov_LotusElize/GamesProject/css/games.css:30]: Border-radius is well supported, so you can remove all these vendor prefixes (`-webkit-border-radius`...`-o-border-radius`)
Please fix in the next task

**Pull request merged**